README.md

```diff
@@ -33,13 +33,11 @@ Read [this issue string](https://github.com/OpenHumans/open-humans/issues/690) f
 
 **Django tutorial:** If you're new to Django, we recommend you do [the Django tutorial](https://docs.djangoproject.com/en/2.0/intro/tutorial01/) first, to get a feel for Django project development
 
-**Virtual environments:** If you're new to developing Python projects, we strongly recommend you learn about using virtual environments for your work.
-  This project uses pipenv — the officially recommended Python packaging tool from Python.org. Read more about pipenv here: https://docs.pipenv.org/
+**Virtual environments:** If you're new to developing Python projects, we strongly recommend you learn about using virtual environments for your work. Either virtualenv (with virtualenvwrapper recommended) or conda are reasonable options.
+ * About virtualenv and virtualenvwrapper: http://docs.python-guide.org/en/latest/dev/virtualenvs/
+ * About Conda's virtual environments: https://conda.io/docs/user-guide/tasks/manage-environments.html
 
-  * Install pipenv with `pip install pipenv`
-  * Activate the virtual environment with `pipenv shell`
-
-  When developing on your own machine, you should be installing requirements from `Pipfile` within a virtual environment – not globally.
+When developing on your own machine, you should be installing requirements from `requirements.txt` within a virtual environment – not globally.
```

Comment:
We recently switched from `virtualenv` to `pipenv` in this PR: https://github.com/OpenHumans/oh-proj-management/pull/9 so we would like to keep the pipenv instructions